docs/configuration.rst

```diff
@@ -470,6 +470,17 @@ below:
      default: "DES,Blowfish,RC2,RC4,IDEA"
      description: Weak cipher algorithms that should be reported, comma separated.
 
+   DD_UNLOAD_MODULES_FROM_SITECUSTOMIZE:
+     type: String
+     default: "0"
+     description: |
+        Controls whether module cloning logic is executed by ``sitecustomize.py``. Module cloning involves saving copies of dependency modules for internal use by ``ddtrace``
```

Comment:
```suggestion
        Controls whether module cloning logic is executed by ``ddtrace-run``. Module cloning involves saving copies of dependency modules for internal use by ``ddtrace``
```

Reply:
`ddtrace-run` is what customers are aware of, `sitecustomize.py` is a little bit of internal details that most might not know about (?)